app/kumactl/cmd/install/testdata/install-control-plane.cni-enabled.golden.yaml

```diff
@@ -1977,7 +1981,7 @@ spec:
       containers:
         - name: install-cni
           image: "docker.io/kumahq/install-cni:0.0.10"
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
why are we changing the old image policy? I think that `IfNotPresent` is correct but why was it `Always` before?

Reply:
I don't know, but it was just wrong, and now can be configured if you want (wasn't possible before)